profiles/base/deployment.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: deployment
+  name: profiles-deployment
```

Comment:
Why are you adding profiles as a prefix? Doesn't kustomization.yaml take care of that?